trans_sec/controller/ddos_sdn_controller.py

```diff
@@ -63,6 +63,7 @@ def start(self):
         logger.info('Starting Controllers')
         for controller in self.controllers.values():
             controller.start()
+            controller.switch_forwarding()
```

Comment:
Add this call into abstract_controller#start() after line 61 after self.__add_helpers.